src/server/game/Entities/AreaTrigger/AreaTrigger.h

```diff
@@ -21,11 +21,14 @@
 #include "Object.h"
 #include "Position.h"
 #include "Spline.h"
+#include "AreaTriggerAI.h"
```

Comment:
It's the only  '#include "AreaTriggerAI.h"' left, remove it won't break non-PCH build ?